src/test/java/bio/terra/service/dataset/DatasetJsonConversionTest.java

```diff
@@ -20,19 +22,28 @@
 import bio.terra.model.DatasetRequestAccessIncludeModel;
 import bio.terra.model.DatasetRequestModel;
 import bio.terra.model.DatasetSpecificationModel;
+import bio.terra.model.SnapshotBuilderSettings;
 import bio.terra.model.TableDataType;
 import bio.terra.model.TableModel;
 import bio.terra.service.resourcemanagement.MetadataDataAccessUtils;
 import bio.terra.service.resourcemanagement.google.GoogleProjectResource;
+import bio.terra.service.snapshotbuilder.SnapshotBuilderService;
 import java.time.Instant;
 import java.util.Collections;
 import java.util.List;
 import java.util.UUID;
 import org.junit.Before;
 import org.junit.Test;
 import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.mock.mockito.MockBean;
 import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringRunner;
 
+@RunWith(SpringRunner.class)
+@ContextConfiguration(classes = {DatasetJsonConversion.class})
 @ActiveProfiles({"google", "unittest"})
 @Category(Unit.class)
```

Comment:
Could you migrate this test to JUnit 5? IntelliJ has a command to do it that should do a lot for you. You'll also want to set up the test service in the Before method:
```
@ExtendWith(MockitoExtension.class)
@ActiveProfiles({"google", "unittest"})
@Tag(Unit.TAG)
class DatasetJsonConversionTest {
...
  private DatasetJsonConversion datasetJsonConversion;

  @Mock private SnapshotBuilderService snapshotBuilderService;
...
  public void setUp() throws Exception {
    datasetJsonConversion = new DatasetJsonConversion(snapshotBuilderService);
    Column datasetColumn =
...
```

Actually, I think you want to use `MockitoExtension` even if you don't move to JUnit 5.